### ISSUE TITLE
QA: Data flow

### ISSUE THREAD

**Issue by alanmoo** [mozilla/donate-wagtail#253]
# Blocked by #115 

Master QA ticket for data capture: Ensuring appropriate information gets into Salesforce

If an item is not completely passing, file a new ticket and edit this post to link to it. For example:
`- [ ] A thing that should work, but doesn't (#1234)`

Any QA tickets that don't show a reference from a master issue will be closed.

- [ ] Paypal donations
  - [x] One time #455 , #488
  - [x] Recurring upgrade #455 , #488, #459
  - [x] Recurring first payment #456, #488
  - [ ] Recurring second payment+ #467
- [ ] Credit card donations
  - [x] One time #455 , #488
  - [x] Recurring upgrade #455 , #488, #459
  - [x] Recurring first payment #456, #488
  - [ ] Recurring second payment+ #467 
  - [x] Address is sent to Basket, even if it's not caught there yet
- [x] Paypal campaign donation (addition of "campaign" field) #457, #468 
  - [x] First time
  - [x] Recurring payment (assuming this should have a campaign field still attached- I'll confirm and update this) (From Steph and Will, the campaign field should not be attached for subsequent recurring payments)
- [x] Credit card campaign donation (same addition) #480 
  - [x] First time
  - [x] Recurring payment (same question as above)
